docs/how-to/manage-identities.md

```diff
@@ -0,0 +1,127 @@
+
+To extend this to additional users, add named identities using the [`add-identities`](../reference/cli-commands/add-identities.md) command with a YAML file configuring the details. For example, to add a new admin "bob" with UID 42 and a new read user "alice" with UID 2000, prepare this file:
+
+```yaml
+# idents-add.yaml
+identities:
+    bob:
+        access: admin
+        local:
+            user-id: 42
+    alice:
+        access: read
+        local:
+            user-id: 2000
+```
+
+and run the following command:
+
+```{terminal}
+   :input: pebble add-identities --from idents-add.yaml
+Added 2 new identities.
+```
+
+To remove existing identities, use [`remove-identities`](../reference/cli-commands/remove-identities.md)with a YAML file that has a `null` value for each identity you want to remove. For example, to remove "alice", prepare this file:
+
+```yaml
+# idents-remove.yaml
+identities:
+    alice: null
+```
+
+and run the following command:
+
+```{terminal}
+   :input: pebble remove-identities --from idents-remove.yaml
+Removed 1 identity.
+```
+
+To update existing identities, use [`update-identities`](../reference/cli-commands/update-identities.md). For example, prepare this file:
+
+```yaml
+# idents-update.yaml
+identities:
+    bob:
+        access: admin
+        local:
+            user-id: 1042
+```
+
+and run the following command:
+
+```{terminal}
+   :input: pebble update-identities --from idents-update.yaml
+Updated 1 identity.
+```
+
+You can use the `--replace` flag to idempotently add or update (or even remove) identities, whether or not they exist. The replace option is useful in automated scripts. For example, to update "bob", add "alice", and remove "mallory" (if it exists), prepare this file:
+
+```yaml
+# idents-replace.yaml
+identities:
+    bob:
+        access: admin
+        local:
+            user-id: 1042
+    alice:
+        access: read
+        local:
+            user-id: 3000
+    mallory: null
+```
+
+and run the following command:
+
+```{terminal}
+   :input: pebble update-identities --from idents-replace.yaml --replace
+Replaced 3 identities.
+```
+
+You can list identities with the [`identities`](../reference/cli-commands/identities.md) command:
+
+```{terminal}
+   :input: pebble identities
+Name   Access  Types
+alice  read    local
+bob    admin   local
+```
+
+Use `--format yaml` (or `--format json`) to show all non-secret fields in the YAML (or JSON) format:
+
+```{terminal}
+   :input: pebble identities --format=yaml
+identities:
+    alice:
+        access: read
+        local:
+            user-id: 3000
+    bob:
+        access: admin
+        local:
+            user-id: 1042
+```
+
+To show a single identity in YAML format, use the `identity` command:
+
+```{terminal}
+   :input: pebble identity alice
+access: read
+local:
+    user-id: 2000
+```
+
+## Seeding initial identities
```

Comment:
This is the only subsection in this how-to doc, it feels not so good. Maybe we can create more sections, for example:

```yaml
# How to manage identities

painting the picture

## Create identities
### add-identities
### Seeding initial identities

## View identities
### list all
### list single
### output format

## Update identities
### update
### remove
### idempotency

## Read more

some more links
```

Reply:
I agree a bit more structure would be helpful to navigate the available content here.

Reply:
Added